support/cas-server-support-ldap/src/main/java/org/apereo/cas/authentication/LdapAuthenticationHandler.java

```diff
@@ -211,7 +225,7 @@ protected Principal createPrincipal(final String username, final LdapEntry ldapE
                         key, ldapEntry.getDn());
             }
         });
-        final String dnAttribute = getName().concat(".").concat(id);
+        final String dnAttribute = (StringUtils.isNotBlank(this.principalDnAttributeName)) ? this.principalDnAttributeName : principalDnAttributeName;
```

Comment:
Are you sure this line makes sense?

Reply:
Of course ... not!
Sorry about that. It's fixed now